concent_api/requirements.lock

```diff
@@ -20,29 +21,35 @@ cytoolz==0.9.0.1
 dask==0.18.1
 decorator==4.3.0
 Django==1.11.13
-django-constance[database]==2.2.0
-eth-abi==0.5.0
-eth-keyfile==0.4.1
-eth-keys==0.1.0b4
-eth-tester==0.1.0b15
-eth-utils==0.7.4
+django-constance==2.2.0
```

Comment:
Why is the dependency on `django-constance` being changed here?

Reply:
As we talked recently on `slack` and you decided that if something is in `pip freeze` it should be also in `requirements.lock` This is how `django-constance[database]` is installed by `pip`: 
```
django-constance==2.2.0
django-picklefield==1.0.0
```

Reply:
But it's completely unrelated to this pull request. You should make this change separately.

Reply:
Extracted from this PR